mozilla_django_oidc/middleware.py

```diff
@@ -0,0 +1,82 @@
+try:
+    from urllib import urlencode
+except ImportError:
+    from urllib.parse import urlencode
+
+import time
+
+import django
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.utils.crypto import get_random_string
+
+from mozilla_django_oidc.utils import absolutify, import_from_settings
+
+
+# Django 1.11 makes changes to how middleware work. In Django 1.11+, we want to
+# use the mixin so that our middleware works as is.
+if tuple(django.VERSION[0:2]) >= (1, 10):
```

Comment:
First of all, `django.VERSION[0:2]` is already a tuple. And you don't need the slice. 
This is the exact same meaning: `if django.VERSION >= (1, 10):` and it's neater. 

Secondly, the *comment* above says Django 1.11 but this if statement is about Django 1.10.